src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package duke.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that has deadline.
+ */
+public class Deadline extends Task {
+    protected LocalDateTime by;
```

Comment:
I think it might suit the coding standards better if this was a noun! e.g. `dueDate`